### PR TITLE
Increase page runtime timeout to 60 seconds as some stories take more than 30 seconds

### PR DESCRIPTION
**Pull request by sunilsurana**
Increase page runtime timeout to 60 seconds as some stories take more than 30 seconds

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            ],
+            "cwd": "C:\\GIT\\vrt",
+            "program": "C:\\GIT\\storywright\\bin\\storywright.js",
+            "args": ["--concurrency","1","--browsers","chromium","--destpath","chromium-baseline"],
+            "outFiles": [
+                "${workspaceFolder}/lib/**/*.js"
+            ]
+        }
+    ]
+}
```
Comment (p01) [inline .vscode/launch.json:22]: That is beyond the scope of the PR

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "pwa-node",
+            "request": "launch",
+            "name": "Launch Program",
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "cwd": "C:\\GIT\\vrt",
```
Comment (p01) [inline .vscode/launch.json:14]: Why the hardcoded path instead of `"cwd": "${worksapceRoot}/something",` ?

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            "program": "C:\\GIT\\storywright\\bin\\storywright.js",
```
Comment (p01) [inline .vscode/launch.json:15]: `"program": "${worksapceRoot}/bin/storywright.js",`

File: .vscode/launch.json (hunk 1 above)
Comment (sunilsurana) [reply .vscode/launch.json]: Delete the file for now

**Pull request merged**